src/operators.jl

```diff
@@ -137,6 +137,23 @@ function union{T<:SimpleGraph}(g::T, h::T)
     return r
 end
 
+
+"""Merges graphs `g` and `h`. The resulting graph has `nv(g)+nv(h)``
+    and ne(g)+ne(h) edges. Vertex indexes in `h` are shifted by `nv(g)``.
+"""
+function join{T<:SimpleGraph}(g::T, h::T)
```

Comment:
I like the name because it matches the corresponding operation on the adjacency matrix.